sys/test_utils/print_stack_usage/print_stack_usage.c

```diff
@@ -20,11 +20,37 @@
 #include "thread.h"
 #include "log.h"
 
+#if MODULE_FMT
+#include "fmt.h"
+#endif
+
+#if MODULE_FMT
+/* fmt's `print_str()` needs very little stack. ~200 total was fine on Cortex-M. */
+# define MIN_SIZE   (256)
+#else
+# define MIN_SIZE   (THREAD_EXTRA_STACKSIZE_PRINTF)
```

Comment:
```suggestion
# define MIN_SIZE   (THREAD_STACKSIZE_TINY + THREAD_EXTRA_STACKSIZE_PRINTF)
```

My interpretation always was that any thread needed at least `THREAD_STACKSIZE_TINY` to be usable at all. And that `THREAD_EXTRA_STACKSIZE_PRINTF` has to come in addition if a thread with a tweaked stack size now wants to start using `printf()`.

Reply:
that does make sense. but, at that point the current extra_stacksize_printf is too large, at least on cortex-m.

Reply:
But that is probably another construction zone. :)